math/quadratic_equations_complex_numbers.cpp

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file
+ * @brief Calculate quadratic equation with complex roots, i.e. b^2 - 4ac < 0.
+ * @see https://en.wikipedia.org/wiki/Quadratic_equation,
+ * https://en.wikipedia.org/wiki/Quadratic_equation#discriminant
+ *
+ * @author [Renjian-buchai](https://github.com/Renjian-buchai)
+ */
+
+#include <array>
+#include <cassert>
+#include <cmath>
+#include <complex>
+#include <exception>
+#include <iomanip>
+#include <iostream>
```

Comment:
```suggestion
#include <array>         /// for arrays
#include <cassert>      /// for assert
#include <cmath>       /// for common mathematical operations
#include <complex>    /// for complex number mathematics
#include <exception>  /// for exceptions 
#include <iomanip>    /// for I/O parametric manipulations 
#include <iostream>   /// for I/O operations
```
Should add explanations for including headers and libraries